src/algebra/quaternion.lean

```diff
@@ -573,3 +585,41 @@ monoid_with_zero_hom.map_div norm_sq a b
 end field
 
 end quaternion
+
+namespace cardinal
+
+open_locale cardinal quaternion
+
+/-- A useful lemma for the following theorems. -/
+lemma pow_four {A : Type*} : #A * #A * #A * #A = #A ^ 4 :=
+by ring
```

Comment:
... except you don't actually use this theorem!

Reply:
Well, I figured it might be useful for anyone else who wants to use the theorems below. I can delete if you think otherwise.

Reply:
I think probably best to just remove it. If we did have it, we should put it next to `pow_two`, but we don't seem to have `pow_three` either.